src/modules/aws.rs

```diff
@@ -212,10 +231,12 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
         return None;
     }
 
-    // only display if credential_process is defined or has valid credentials
+    // only display if credential_process, source_profile are defined or has valid credentials
```

Comment:
// only display in the presence of credential_process, source_profile or valid credentials